README.rst

```diff
@@ -95,6 +242,8 @@ Updating a Replication Set
 Dropping a Replication Set
 --------------------------
 
+Replication sets may be dropped via the ``drop`` command.
```

Comment:
What's the consequence of dropping a replication set? Any effect on the relay for it?